genesyscloud/tfexporter/tf_exporter_resource_test.go

```diff
@@ -32,187 +32,188 @@ type registerTestInstance struct {
+	providerResources["genesyscloud_routing_email_route"] = gcloud.ResourceRoutingEmailRoute()
+	providerResources["genesyscloud_routing_language"] = gcloud.ResourceRoutingLanguage()
+	providerResources["genesyscloud_routing_queue"] = gcloud.ResourceRoutingQueue()
+	providerResources["genesyscloud_routing_skill"] = gcloud.ResourceRoutingSkill()
+	providerResources["genesyscloud_routing_settings"] = gcloud.ResourceRoutingSettings()
+	providerResources["genesyscloud_routing_utilization"] = gcloud.ResourceRoutingUtilization()
+
+	providerResources["genesyscloud_routing_wrapupcode"] = gcloud.ResourceRoutingWrapupCode()
+	providerResources["genesyscloud_telephony_providers_edges_did_pool"] = gcloud.ResourceTelephonyDidPool()
+	providerResources["genesyscloud_telephony_providers_edges_edge_group"] = gcloud.ResourceEdgeGroup()
+	providerResources["genesyscloud_telephony_providers_edges_extension_pool"] = gcloud.ResourceTelephonyExtensionPool()
+	providerResources["genesyscloud_telephony_providers_edges_phone"] = gcloud.ResourcePhone()
+	providerResources["genesyscloud_telephony_providers_edges_site"] = gcloud.ResourceSite()
+	providerResources["genesyscloud_telephony_providers_edges_phonebasesettings"] = gcloud.ResourcePhoneBaseSettings()
+	providerResources["genesyscloud_telephony_providers_edges_trunkbasesettings"] = gcloud.ResourceTrunkBaseSettings()
+	providerResources["genesyscloud_telephony_providers_edges_trunk"] = gcloud.ResourceTrunk()
+	providerResources["genesyscloud_user_roles"] = gcloud.ResourceUserRoles()
+	providerResources["genesyscloud_webdeployments_configuration"] = gcloud.ResourceWebDeploymentConfiguration()
+	providerResources["genesyscloud_webdeployments_deployment"] = gcloud.ResourceWebDeployment()
+	providerResources["genesyscloud_widget_deployment"] = gcloud.ResourceWidgetDeployment()
+
+	providerResources["genesyscloud_outbound_attempt_limit"] = outbound_attempt_limit.ResourceOutboundAttemptLimit()
+	providerResources["genesyscloud_outbound_callanalysisresponseset"] = ob.ResourceOutboundCallAnalysisResponseSet()
+	providerResources["genesyscloud_outbound_callabletimeset"] = ob.ResourceOutboundCallabletimeset()
+	providerResources["genesyscloud_outbound_campaign"] = ob.ResourceOutboundCampaign()
+	providerResources["genesyscloud_outbound_contact_list"] = outbound_contact_list.ResourceOutboundContactList()
+	providerResources["genesyscloud_outbound_contactlistfilter"] = ob.ResourceOutboundContactListFilter()
+	providerResources["genesyscloud_outbound_messagingcampaign"] = ob.ResourceOutboundMessagingCampaign()
+	providerResources["genesyscloud_outbound_sequence"] = ob.ResourceOutboundSequence()
+	providerResources["genesyscloud_outbound_dnclist"] = ob.ResourceOutboundDncList()
+	providerResources["genesyscloud_outbound_campaignrule"] = ob.ResourceOutboundCampaignRule()
+	providerResources["genesyscloud_outbound_ruleset"] = obRuleset.ResourceOutboundRuleset()
```

Comment:
`go fmt` made a load of formatting changes, but this was the only line added. Nothing was removed.